src/main/java/com/sap/oss/phosphor/fosstars/data/github/PackageManagement.java

```diff
@@ -83,6 +86,7 @@ public class PackageManagement extends CachedSingleFeatureGitHubDataProvider<Pac
         ".vcxproj"::equals, ".fsproj"::equals, "packages.config"::equals);
     register(RUBYGEMS, "Gemfile.lock"::equals, "Gemfile"::equals, ".gemspec"::endsWith);
     register(COMPOSER, "composer.json"::equals, "composer.lock"::equals);
+    register(GOMODULES, "go.mod"::equals);
```

Comment:
Is this the only form of Go Module possible for Package Management.

Maybe is there a `.lock` file?

Reply:
In addition to go.mod, the go command maintains a file named go.sum containing the expected [cryptographic hashes](https://go.dev/cmd/go/#hdr-Module_downloading_and_verification) of the content of specific module versions. Do we need to check this as well?